tests/openassetio/test/managerValidator/resources/executeSuiteTests.py

```diff
@@ -0,0 +1,82 @@
+#
```

Comment:
[consider][minor]  It would be nice to name this file and its fixtures to better resemble the other files in this directory.  Maybe `customTestSuite` / `fixtures_custom`, or `metaTestSuite` :grin: , or maybe similar to before but suffixed to align more closely with other files, `executeSuiteTestSuite` / `fixtures_executeSuite`... as you often say, naming things is hard!

Reply:
How about:
![image](https://user-images.githubusercontent.com/12052093/149373251-1ce6e07b-a76b-4cb4-8abe-d073cb74056b.png)

Reply:
Yeah much nicer!

Reply:
Implemented [here](https://github.com/TheFoundryVisionmongers/OpenAssetIO/pull/184/commits/1bda909a486d09b3f420260ff7f5238095369a3c), along side the module renames we discussed, [here](https://github.com/TheFoundryVisionmongers/OpenAssetIO/pull/184/commits/621e30af1f197889ee74e0e38e0f9495fbaf258c) and [here](https://github.com/TheFoundryVisionmongers/OpenAssetIO/pull/184/commits/42707bff24c3b959bbc153ea59ecc6ee0ec60fd6).